e2e/client/playwright/templates.spec.ts

```diff
@@ -0,0 +1,113 @@
+    await page.locator(s('template-edit-view')).getByRole('button', {name: 'Save'}).click();
+
+    await expect(page.locator(s('template-content', 'content-template--story 1.1'))).toBeVisible();
+});
+
+test('removing template', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/templates');
+
+    await page.locator(s('template-content', 'content-template--story 2', 'template-actions')).click();
+    await page.locator(s('template-actions--options')).getByRole('button', {name: 'Remove'}).click();
+    await page.locator(s('modal-confirm')).getByRole('button', {name: 'Ok'}).click();
+    await expect(page.locator(s('template-content', 'content-template--story 2'))).not.toBeVisible();
+});
+
+test('Assign template to a desk', async ({page}) => {
+    const monitoring = new Monitoring(page);
+
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/workspace/monitoring');
+    await monitoring.selectDeskOrWorkspace('Finances');
+
+    await page.locator(s('content-create')).click();
+    await page.locator(s('content-create-dropdown')).getByRole('button', {name: 'More Templates...'}).click();
+    await expect(page.locator(s('content-create-dropdown')).getByRole('button', {name: 'Story 2'})).not.toBeVisible();
+
+    // assign template to the desk
+    await page.goto('/#/settings/templates');
+    await page.locator(s('template-content', 'content-template--story 2', 'template-actions')).click();
+    await page.locator(s('template-actions--options')).getByRole('button', {name: 'Edit'}).click();
+    await page.locator(s('template-edit-view', 'desks', 'desk--Finances')).first().click();
+
+    await page.goto('/#/workspace/monitoring');
+    await monitoring.selectDeskOrWorkspace('Sports');
+
+    await page.locator(s('content-create')).click();
+    await page.locator(s('content-create-dropdown')).getByRole('button', {name: 'More Templates...'}).click();
+    await expect(page.locator(s('content-create-dropdown')).getByRole('button', {name: 'Story 2'})).toBeVisible();
+});
+
+test('Default content template', async ({page}) => {
+    const monitoring = new Monitoring(page);
+
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/desks');
+
+    await page.locator(s('desk--Sports', 'desk-actions')).click();
+    await page.locator(s('actions--options')).getByRole('button', {name: 'Edit'}).click();
+    await page.locator(s('desk-config-modal', 'field--default-content-template')).selectOption({label: 'story 2'});
+    await page.locator(s('desk-config-modal')).getByRole('button', {name: 'done'}).click();
+
+    await page.goto('/#/workspace/monitoring');
+    await monitoring.selectDeskOrWorkspace('Sports');
+    await page.locator(s('content-create')).click();
+    await expect(page.locator(s('content-create-dropdown', 'default-desk-template'))).toHaveText('story 2');
+});
+
+test('Prefill template', async ({page}) => {
```

Comment:
Review naming of all newly added tests in this PR. The name should start with a lowercase letter and it should read well when `test` function is read as a part of the test name. Also make test names more descriptive so it can be understood better by people who are not too familiar with the feature. For example this could be more clearly phrased `test new article prefilling with content set in template`